### PR TITLE
Update Pbsrc.com.xml

### LINKED ISSUE #3527: Pbsrc.com STILL breaks photobucket functionality
This was reported as an issue in the 2015.8.13 version and is still an outstanding issue in 2015.12.16.

Pbsrc.com breaks flash copy action in the Email & IM, Direct, HTML, IMG fields:
http://s721.photobucket.com/user/terrorist96/media/ghgdsfghfd_zps0378f4b9.png.html

### PR DESCRIPTION
**Pull request by terrorist96**
Fixes #3527 
@J0WI

### CODE REVIEW

**Comment by terrorist96** [comment]
Okay, I don't know what's still causing it to fail. Any help please?

File: src/chrome/content/rules/Pbsrc.com.xml [hunk 1]
```diff
@@ -51,7 +51,9 @@
 
 -->
 <ruleset name="Pbsrc.com (partial)">
-
+	
+	<!-- https://github.com/EFForg/https-everywhere/issues/3527 -->
+	<exclusion pattern="^http://pic2\.pbsrc\.com/flash/ZeroClipboardFV3\.swf">
```
Comment (J0WI) [inline src/chrome/content/rules/Pbsrc.com.xml]: `>` should be ` />`

**Comment by terrorist96** [comment]
You edited your post. 😝 
Fixed now.

**Comment by terrorist96** [comment]
Lol now what?? 😡

**Comment by J0WI** [comment]
You need to find a valid url to fix the test (or fix the rule if it's broken).

**Comment by terrorist96** [comment]
I thought you gave me the test url?
Would the original report link be an ok test url?
http://s721.photobucket.com/user/terrorist96/media/ghgdsfghfd_zps0378f4b9.png.html

I can't find a way to directly link to the SHARE THIS PHOTO element on the right side.

**Comment by J0WI** [comment]
Please read the outut from Travis.
https://travis-ci.org/EFForg/https-everywhere/jobs/130376145#L616

You need a valid resource to test the redirect from `static2.pbsrc.com` to `ostatic2.pbsrc.com`.
To work around the 403 warning you should also exclude the root of this host:
`<exclusion pattern="^http://static2\.pbsrc\.com/$" />`

**Comment by J0WI** [comment]
All test passed.

**Comment by terrorist96** [comment]
@J0WI this is still broken for me in 2016.6.9

**Comment by terrorist96** [comment]
Looks to be fixed now.

**Comment by terrorist96** [comment]
http://media.photobucket.com/user/ThomasLovesJeremy/media/tumblr_m2lvj8gtln1qhzv6oo1_500.gif.html?filters[term]=animated%20gifs&filters[primary]=images&filters[featured_media]=1748&filters[secondary]=videos&sort=1&o=2

**Pull request merged**